### ISSUE TITLE
Select Width Issue

### ISSUE THREAD

**Issue by ayushsharma82** [geist-org/geist-ui#786]
# Bug report 🐞

## Version & Environment

  - Version of browser - Safari - Latest
  - Version of `geist-ui/core`

## Expected Behaviour

The width should be enforced when set through `width` property, but the component doesn't respect the value. ( i.e it doesn't decrease its width after achieving a certain minimum width ).

## Actual results (or Errors)

The width can only be increased but stops decreasing after a certain minimum width.

<img width="220" alt="image" src="https://user-images.githubusercontent.com/26592968/173575699-184021e1-6647-44cf-a9a6-988640de8583.png">

**Comment by ayushsharma82** [comment]
Snippet for reproducing the error:

```js
const [Limit, setLimit] = useState("10");

<Select placeholder="Show" value={Limit} onChange={(v) => setLimit(v)} scale={0.5}>
       <Select.Option value="10">10</Select.Option>
       <Select.Option value="25">25</Select.Option>
       <Select.Option value="50">50</Select.Option>
       <Select.Option value="75">75</Select.Option>
       <Select.Option value="100">100</Select.Option>
</Select>

```

**Comment by hothoney** [comment]
I don't think this is a bug, but I think it's unnessecsary to have a minimum with for Select component, I'd like to fix it if it's OK. @unix
